save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/ExecutionView.kt

```diff
@@ -87,10 +96,20 @@ external interface StatusProps<D : Any> : TableProps<D> {
      */
     var status: TestResultStatus?
 
+    /**
+     * Test Result Status to filter by
+     */
+    var testName: String?
+
     /**
      * Name of test suite
      */
     var testSuite: String?
+
+    /**
+     * Test Result Status to filter by
+     */
+    var tag: String?
```

Comment:
Maybe we can wrap these four filters into a data class? So that state has only one additional field instead of four

Reply:
Done